src/hato/multipart.clj

```diff
@@ -21,7 +24,11 @@
   [{:keys [content content-type]}]
   (str "Content-Type: "
        (cond
-         content-type content-type
+         content-type (if (string? content-type)
+                        content-type
+                        (str (:mime-type content-type)
+                             (when-let [charset (:charset content-type)]
+                               (str "; charset=" charset))))
```

Comment:
There is an issue today where if you supply a String, we essentially only ever allow UTF-8. This allows the content-type param to be either a string or map which can describe the mime type and charset. I could see an argument for this being another key but this approach more closely aligns with how libraries like Apache HttpClient represent ContentType.